src/utils/hooks/useListTemplates.tsx

```diff
@@ -31,7 +31,7 @@ const useListTemplates = (templatePermissions: TemplatePermissions, isLoading: b
 
   const { data, error: apolloError } = useGetTemplatesQuery({
     skip: isLoading,
-    fetchPolicy: 'network-only',
+    // fetchPolicy: 'network-only',
```

Comment:
So there was a bug where if you loaded the app in a new tab with valid auth credentials already in storage, then the Dashboard would load but wouldn't fetch any templates. I had a good poke around in here, and it's really weird because the `data` in this hook was showing `undefined`, yet I could clearly see the query in the network tab was returning correct results. 🤷‍♂️

Anyway, the problem goes away by turning this off, no idea why. I'm sure it's fine, there's no critical, updated data here that requires a non-cached response every time.

Reply:
I couldn't replicate this error.  I tried 'no-cache' as well, which I don't think is what you want long term anyway, but I always got data....  So I guess we'll leave it.  It'll be fine....